### PR TITLE
fix(rolling upgrade version): bumping version to last OSS

### PR DESCRIPTION
**Pull request by fgelcer**
after 3.3 was released, and now working on branch 4.0, bumping up
the base version to run on 3.3 (last released was 3.3.4).

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I gave variables/functions meaningful self-explanatory names
- [ ] I didn't leave commented-out/debugging code
- [ ] I didn't copy-paste code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: jenkins-pipelines/rolling-upgrade-centos7.jenkinsfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ rollingUpgradePipeline(
     params: params,
 
     backend: 'gce',
-    base_versions: ['2019.1', '3.2'],
```
Comment (roydahan) [inline jenkins-pipelines/rolling-upgrade-centos7.jenkinsfile]: This is wrong, we should have here 2019.1

File: jenkins-pipelines/rolling-upgrade-ubuntu16.04.jenkinsfile [hunk 2]
```diff
@@ -7,7 +7,7 @@ rollingUpgradePipeline(
     params: params,
 
     backend: 'gce',
-    base_versions: ['2019.1', '3.2'],
```
Comment (roydahan) [inline jenkins-pipelines/rolling-upgrade-ubuntu16.04.jenkinsfile]: same as above

File: jenkins-pipelines/rolling-upgrade-ubuntu18.04.jenkinsfile [hunk 3]
```diff
@@ -7,7 +7,7 @@ rollingUpgradePipeline(
     params: params,
 
     backend: 'gce',
-    base_versions: ['2019.1', '3.2'],
```
Comment (roydahan) [inline jenkins-pipelines/rolling-upgrade-ubuntu18.04.jenkinsfile]: same as above

File: jenkins-pipelines/rolling-upgrade-centos7.jenkinsfile (hunk 1 above)
Comment (fgelcer) [reply jenkins-pipelines/rolling-upgrade-centos7.jenkinsfile]: but we have

**Comment by roydahan** [comment]
Unless there is a logic somewhere that picks OSS to OSS base version, We shouldn't have this 2019.1.
@amoskong do I miss something?

**Comment by fgelcer** [comment]
@roydahan , i don't understand what you mean by your comment on the original line, and not in the new line...

**Comment by fruch** [comment]
> Unless there is a logic somewhere that picks OSS to OSS base version, We shouldn't have this 2019.1.
> @amoskong do I miss something?

we have this logic:
https://github.com/scylladb/scylla-cluster-tests/blob/master/vars/supportedUpgradeFromVersions.groovy#L3

i.e. we remove the enterprise versions from the list, when running on open-source.
the idea was to minimize the diff we need to make on branches, i.e. enterprise ones.

File: jenkins-pipelines/rolling-upgrade-centos7.jenkinsfile (hunk 1 above)
Comment (amoskong) [reply jenkins-pipelines/rolling-upgrade-centos7.jenkinsfile:10]: The change is correct.

**Comment by amoskong** [comment]
> Unless there is a logic somewhere that picks OSS to OSS base version, We shouldn't have this 2019.1.
> @amoskong do I miss something?

The '2019.1' will be filtered out in opensource upgrade testing.
So we can remove it from opensource version.

File: jenkins-pipelines/rolling-upgrade-centos7.jenkinsfile (hunk 1 above)
Comment (amoskong) [reply jenkins-pipelines/rolling-upgrade-centos7.jenkinsfile:10]: The enterprise version will be filtered out in pipeline on opensource upgrade jobs.
Actually it doesn't effect, we can remove it.

**Comment by fgelcer** [comment]
@fruch , @bentsi , please merge and  back port ASAP.
@roydahan FYI

**Pull request merged**